policy/release/lib/attestations.rego

```diff
@@ -51,31 +50,34 @@ pipelinerun_attestations := att if {
 
 pipelinerun_slsa_provenance02 := [statement |
 	some att in input.attestations
-	statement := lib.statement(att)
-	statement.predicate.buildType in pipelinerun_att_build_types
+	statement := att
```

Comment:
There is a change in behavior here. Not sure if it is intentional. Previously, `pipelinerun_slsa_provenance02` would not contain the "wrapped" objects. Now it does.

Previously:

```
[{"_type": "...", "predicate": ...}]
```

Now:

```
[{"statement": {"_type": "...", "predicate": ...}}]
```

There is value in having the code base always dealing with the "wrapped" object. But there is also the downside for someone new to the project that they are not dealing with the slsa provenance attestation directly. I welcome the new behavior, but want to make sure this is indeed intentional.

Same for `pipelinerun_slsa_provenance_v1` and `taskrun_attestations`.

Reply:
This wasn't an intentional change, but I do think it's worth having a conversation about. I think it's easier to document the `statement` wrapper for newbies than keep track of where we need to treat the `statement` wrapper in a special way across the entire codebase.

But if we anticipate end-users (e.g. custom policy authors) interacting with this stuff routinely, maybe it's worth dealing with the pain for end-user benefit...

Reply:
Let's favor consistency on the code base over a hypothetical lowered entry barrier.